packages/eslint-config-terra/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Locked `eslint-plugin-formatjs` to `2.19.1` as it has a peerDependency of `eslint@7` and no longer conflicts with `eslint-config-terra`.
```

Comment:
Can we reword this to something like the following so that it is clear that the change is so that both `eslint-plugin-formatjs` and `eslint-config-terra` have the same peer dependency version of `eslint`?

```suggestion
  * Locked `eslint-plugin-formatjs` to `2.19.1` so that the version of its peerDependency `eslint@7` is matches that in `eslint-config-terra`.
```